modules/ROOT/partials/events/table-events.adoc

```diff
@@ -1,10 +1,10 @@
-The following events are provided by the xref:table.adoc[Table plugin].
+The following events are provided by the core table functionality.
 
 [cols="1,1,2",options="header"]
 |===
 |Name |Data |Description
-|newrow |N/A |Fired when a row is created
-|newcell |N/A |Fired when a cell is created
+|NewRow |N/A |Fired when a row is created
+|NewCell |N/A |Fired when a cell is created
 |ObjectResizeStart |N/A |Fired when a resize action is started on a table, row, column or cell using the resize bars
 |ObjectResized |N/A |Fired when a resize action is finished on a table, row, column or cell using the resize bars
 |TableModified |`+{ table: HTMLTableElement, structure: boolean, style: boolean }+` |Fired when style or structural changes are made to a table.
```

Comment:
This entire bit of information is duplicated in core... I think it's rather confusing having this and can be detrimental to search to have things duplicated. We might need to have a team chat about what to do with this. Also sorry I missed this in the draft version yesterday 😞

Reply:
Yeah, I wasn't super happy with having it duplicated. The part that complicates it is that ObjectResizeStart and ObjectResized are used by more than just the table plugin and are already in the core events table so linking directly to the events.adoc page is not simple in its current configuration. Ideally, the core table events would be their own table like the core table commands

Reply:
Do we even need to link to it though from the table plugin? As the table plugin is just about UI components now so to me it doesn't make sense to have it there anyways.

Reply:
I think it's helpful to point out that there's more table related stuff in TinyMCE that is relevant, especially for integrators upgrading. 

Maybe we could leave the duplication for now, and log this as something to follow up later, figure out how to architect this info? Brian would prob have opinions.

Reply:
Good point. Rather than overcomplicating, I have just removed all of the references to table plugin events. This includes in the advtable page as well. 

Also, I removed `Query command values` from the table plugin page as well since it was just a sentence saying to go to the core commands page. Happy to revert that if it makes sense for that still be noted in the table plugin page